syntheticsclientv2/get_checksv2.go

```diff
@@ -46,11 +47,27 @@ func (c Client) GetChecksV2(params *GetChecksV2Options) (*ChecksV2Response, *Req
 	if params.PerPage == 0 {
 		params.PerPage = int(50)
 	}
+	if params.SchedulingStrategy == "" {
+		params.SchedulingStrategy = ""
+	}
 
 	// Make the request
 	details, err := c.makePublicAPICall(
 		"GET",
-		fmt.Sprintf("/tests?testType=%s&page=%d&perPage=%d&orderBy=%s&search=%s", params.TestType, params.Page, params.PerPage, params.OrderBy, params.Search),
+		fmt.Sprintf("/tests?testType=%s&page=%d&perPage=%d&orderBy=%s&search=%s%s&schedulingStrategy=%s%s%s%s%s%s",
+			params.TestType,
+			params.Page,
+			params.PerPage,
+			params.OrderBy,
+			params.Search,
+			activeQueryParam(params.active),
```

Comment:
Please check out these Github Action failures from the CI Checks job.

Looks like mostly just some type checks that need to be cleaned up.

Reply:
Sounds good. I will clean these up